src/m2n/tests/PointToPointCommunicationTest.cpp

```diff
@@ -108,12 +108,12 @@ void runP2PComTest1(const TestContext &context, com::PtrCommunicationFactory cf)
     c.send(data);
     c.receive(data);
 
-    BOOST_TEST(data == expectedData);
```

Comment:
Could we prevent the utilization of the 'old* equal method?

Reply:
Not sure if this is possible or even reasonable. We still need the normal `==` at many places and for some datatypes it is also totally fine to use this operator. I would say it's the job of the programmer to make sure to not compare floating point values via `==`, since this can work, but is generally unsafe.